retrofit/src/main/java/retrofit2/Response.java

```diff
@@ -23,9 +23,12 @@
 import okhttp3.ResponseBody;
 import org.checkerframework.checker.mustcall.qual.*;
 import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.dataflow.qual.*;
 import org.checkerframework.common.returnsreceiver.qual.This;
 
+//This class holds resources but doesn't implement closable or close? These resources must still be closed.
```

Comment:
What method does call `close()` on the relevant field? That method should be the `@MustCall` method of this class - `@MustCall` takes an argument specifically to handle cases where the method that actually does the closing isn't actually named "close".